docs/Backtracing.md

```diff
@@ -0,0 +1,166 @@
+# Backtracing support in Swift
+
+When things go wrong, it's always useful to be able to get a backtrace
+showing where the problem occurred in your program.
+
+Broadly speaking there are three circumstances where you might want a
+backtrace, namely:
+
+-   Program crashes
+-   Runtime errors
+-   Specific user-defined program events
+
+Historically, Swift has tended to lean on operating system crash
+catching support for the first two of these, and hasn't really provided
+any built-in support for the latter. This is fine for Darwin, where the
+operating system provides a comprehensive system-wide crash catching
+facility; it's just about OK on Windows, which also has system-wide
+crash logging; but it isn't great elsewhere, in particular on Linux
+where a lot of server-side Swift programs currently rely on a separate
+package to provide them with some level of backtrace support when errors
+happen.
+
+## What does Swift now support?
+
+Swift now supports:
+
+-   Automatic crash catching and backtrace generation out of the box.
+-   Built-in symbolication.
+-   A choice of unwind algorithms, including "fast", DWARF and SEH.
+-   Interactive(!) crash/runtime error catching.
+
+Crash catching is enabled by default, and won't interfere with any
+system-wide crash reporters you might be using.
+
+## How do I configure backtracing?
+
+There is an environment variable, `SWIFT_BACKTRACE`, that can be used to
+configure Swift's crash catching and backtracing support. The variable
+should contain a `,`-separated list of `key=value` pairs. Supported keys
+are as follows:
+
+| Key              | Default  |  Meaning                                                                               |
```

Comment:
Would it hurt the rendered result if we shrink the tables horizontally as in the old file to ease reading the raw text?